docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -11,14 +11,16 @@
     "(realtime-monitor-drift-tutor)=\n",
```

Comment:
<ol><li><code>The next step is to deploy the model-monitoring job</code> - i think we should change it to <code>The next step is to deploy the model-monitoring app</code> </li><li>  again here <code>After deploying the jobs they show</code> - need to replace <code>job</code> to <code>app</code>    </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/8410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>